chain/sync.go

```diff
@@ -528,8 +528,16 @@ func blockSanityChecks(h *types.BlockHeader) error {
 
 // Should match up with 'Semantical Validation' in validation.md in the spec
 func (syncer *Syncer) ValidateBlock(ctx context.Context, b *types.FullBlock) error {
+	validationStart := time.Now()
+	defer func() {
+		var durationMillisec = float64(time.Since(validationStart).Round(time.Microsecond).Nanoseconds()) / 1000000.0
```

Comment:
or don't round at all

Reply:
I was rounding because occasionally the precision is high enough that the output prints with scientific notation. I sliced off a few of the sig figures primarily to make parsing easier.